ekm/signer_storage.go

```diff
@@ -47,24 +47,22 @@ type Storage interface {
 	SetEncryptionKey(newKey string) error
 	ListAccountsTxn(r basedb.Reader) ([]core.ValidatorAccount, error)
 	SaveAccountTxn(rw basedb.ReadWriter, account core.ValidatorAccount) error
-
-	BeaconNetwork() beacon.BeaconNetwork
 }
 
 type storage struct {
 	db            basedb.Database
-	network       beacon.BeaconNetwork
+	networkConfig networkconfig.Interface
```

Comment:
if only the beacon network part is used I think it makes sense to pass in only the beacon network.

Reply:
fixed